### ISSUE TITLE
Test shows nothing and date detection

### ISSUE THREAD

**Issue by alexng88** [Belval/CRNN#43]
Hello,
  I have 2 questions:
1.  when I run the command 
python run.py -ex ../samples/ -m ./save/  --test --restore  it only shows the following pic...with 0.  what that means?
2.  Can I use crnn to detect the date like the one I attached?
Thanks
![num4](https://user-images.githubusercontent.com/1286456/56949551-cd323780-6b65-11e9-931c-1a2c24e34267.png)

![error](https://user-images.githubusercontent.com/1286456/56949470-9bb96c00-6b65-11e9-8cd4-18cd6aa136e0.jpg)

**Comment by alexng88** [comment]
I retrained the model and use textgenerator, with loss 0.2. 
then I try to recognize the image 
![reault](https://user-images.githubusercontent.com/1286456/57193436-8e3d2100-6f6d-11e9-9905-87ee4a405ddd.jpg)

the result is totally different, any suggestions?

**Comment by Belval** [comment]
1. It's a bug when there aren't enough sample in the batch, there's a related issue.
2. Yes, but you will have to change the `data_manager.py` file to read the labels from a file as slashes cannot be in filenames.
3. How long did you train, how big was your training set?

**Comment by alexng88** [comment]
date test:
![roi9](https://user-images.githubusercontent.com/1286456/57318350-b9ba3a00-712c-11e9-80a6-abb9cc6dbba4.png)
Recognized result: 2610m8

![041018](https://user-images.githubusercontent.com/1286456/57318399-dc4c5300-712c-11e9-84bb-d93b764fc9de.jpg)
Recognized result: 0amom8

![d](https://user-images.githubusercontent.com/1286456/57318481-043bb680-712d-11e9-870a-804418330f9f.jpg)

Recognized result: 0310tm8

I used your pre-trained weight.
I dont need the slash and I just need to number as accurate as possible, any clues?

**Comment by Belval** [comment]
If you only read dates, you can reduce the number of chars so you won't end up training letters when they can't be found in your images.

As for the slashes, if you don't want the model to predict them you can just remove them from the labels.

**Comment by alexng88** [comment]
Hello, thanks for your reply.
I have changed another way to detect (use object detection methods)
I will re-try again your method soon.

**Comment by Belval** [comment]
Closing for inactivity.
